subsys/zigbee/common/zigbee_helpers.h

```diff
@@ -209,6 +209,21 @@ void zigbee_led_status_update(zb_bufid_t bufid, u32_t led_idx);
  */
 #if defined ZB_ED_ROLE
 void user_input_indicate(void);
+
+/**@brief Function for enabling sleepy behavior of End Device.
+ *
+ *
+ * @note This function is to be used with End Devices only.
+ *
+ *  * @param[in] enable   Boolean value indicating if sleepy behavior
+ *                        should be enabled or disabled.
+ */
+void zigbee_configure_sleepy_behavior(bool enable);
```

Comment:
- I think it is not necessary to put two empty lines before a note, but I'm not an expert in this area.
- It is worth mentioning that, if needed, it has to be called after `ZB_INIT()` (well, this might be tough to document - it is inside system initialization) and before `zigbee_enable()` (this is important).

Reply:
Are you sure that this should be **before** `ZB_INIT`?

Reply:
@note changed